### ISSUE TITLE
Fix singular/plural labels for facet badges

### ISSUE THREAD

**Issue by erinchambers** [CDRH/austen#84]
In [app/views/search/_facet_panel.html.erb](https://github.com/CDRH/austen/pull/71/files/c2874553837a7d70d37f3ecad707ec9c98a16f18#diff-903c77aa6dad5f6a0c549b3296195421b79423b86d506ce16dc917a145e477e8), the badge code inside the `else` statement should be singular ("result," not "results").

**Comment by erinchambers** [comment]
Fixed by #88
